crates/forge/tests/integration/runtime.rs

```diff
@@ -59,34 +59,3 @@ fn cairo_test_cheatcode_error() {
         ),
     );
 }
-
-#[test]
-fn cheatcode_invalid_args() {
-    let test = test_utils::test_case!(indoc!(
-        r"
-            use starknet::testing::cheatcode;
-            use snforge_std::_cheatcode::handle_cheatcode;
-
-            #[test]
-            fn cheatcode_invalid_args() {
-                handle_cheatcode(cheatcode::<'replace_bytecode'>(array![].span()));
-                assert(true,'');
-            }
-        "
-    ));
-
-    let result = run_test_case(&test);
-
-    assert_case_output_contains(
-        &result,
-        "cheatcode_invalid_args",
-        indoc!(
-            r#"
-                "Reading from buffer failed, this can be caused by calling starknet::testing::cheatcode with invalid arguments.
-                    Probably `snforge_std`/`sncast_std` version is incompatible, check above for incompatibility warning.
-                    "
-            "#
-        ),
-    );
-    assert_failed(&result);
-}
```

Comment:
Do we have an issue to re-add these tests somewhere in cairo?

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/2765